Sources/SwiftSyntaxBuilderGeneration/SyntaxBuildableType.swift

```diff
@@ -19,11 +22,6 @@ struct SyntaxBuildableType: Hashable {
   let tokenKind: String?
   let isOptional: Bool
 
-  /// If optional `?`, otherwise the empty string.
-  var optionalQuestionMark: String {
-    isOptional ? "?" : ""
-  }
```

Comment:
This property is replaced with `optionalWrapped(type:)` and `optionalChained(expr:)`, depending on whether a type or an expression is needed.